python/arcticdb/version_store/library.py

```diff
@@ -1338,18 +1338,18 @@ def get_description(self, symbol: str, as_of: Optional[AsOf] = None) -> SymbolDe
             For documentation on each field.
         """
         info = self._nvs.get_info(symbol, as_of)
-
-        last_update_time = pd.to_datetime(info["last_update"])
+        last_update_time = pd.to_datetime(info["last_update"], utc=True)
         columns = tuple(NameWithDType(n, t) for n, t in zip(info["col_names"]["columns"], info["dtype"]))
         index = NameWithDType(info["col_names"]["index"], info["col_names"]["index_dtype"])
+        date_range = tuple(map(lambda x: x.replace(tzinfo=datetime.timezone.utc), info["date_range"]))
 
         return SymbolDescription(
             columns=columns,
             index=index,
             row_count=info["rows"],
             last_update_time=last_update_time,
             index_type=info["index_type"],
-            date_range=info["date_range"],
+            date_range=date_range,
```

Comment:
I think we should, even though this may be a breaking change for some users. We'll have to make it clear in the release notes. 

Can we make them both `datetime.datetime`? Would it change the behaviour of the `_store.py` API?

Reply:
I don't see any potential breaks, however it seems like the conversion cuts the nanoseconds and we get a warning:
```
UserWarning: Discarding nonzero nanoseconds in conversion
```
It we're ok with it I can make it all `datetime.datetime`

Reply:
oh gosh you're right - I remember that's caused an issue before :/

I'm inclined to do a complete 180 then and say let's stick with `pandas.datetime64`. Don't want to lose any precision.

Reply:
I would avoid `datetime.datetime` as it can't hold nanoseconds in some Python versions!